components/automate-ui/src/app/pages/api-token/details/api-token-details.component.ts

```diff
@@ -74,7 +118,8 @@ export class ApiTokenDetailsComponent implements OnInit, OnDestroy {
     this.saveInProgress = true;
     const name: string = this.updateForm.controls.name.value.trim();
     const active = <TokenStatus>this.updateForm.controls.status.value === 'active';
```

Comment:
❓I don't see how this `<TokenStatus>` cast matters here. We're comparing to another string, which happens to be `'active'`, one of the possible `TokenStatus` values (👉 string literal types); but we're building a boolean.
```suggestion
    const active: boolean = this.updateForm.controls.status.value === 'active';
```

Reply:
It does add some compile-time safety, as it turns out. With that present, the compiler disallows misspellings of 'active' !

Reply:
Ah, right. Well, I guess that's fair, a microscopic, but actual, advantage 😉